02-household-queries/ingest.py

```diff
@@ -0,0 +1,97 @@
+from langchain_google_genai import ChatGoogleGenerativeAI, GoogleGenerativeAIEmbeddings
+from langchain.prompts import PromptTemplate
+from langchain_community.llms import GPT4All
+from bs4 import BeautifulSoup
+from langchain_community.document_loaders import JSONLoader, PDFMinerLoader
+from langchain.docstore.document import Document
+from langchain_text_splitters import RecursiveCharacterTextSplitter
+from langchain.chains.combine_documents import create_stuff_documents_chain
+from langchain.chains import create_retrieval_chain, LLMChain, RetrievalQA
+from langchain_community.vectorstores import Chroma
+from langchain.embeddings.sentence_transformer import SentenceTransformerEmbeddings
+import json
+import os
+from pathlib import Path
+from pprint import pprint
+import dotenv
+import requests
+
+dotenv.load_dotenv()
+
+# Load the models
+GOOGLE_API_KEY = os.environ.get('GOOGLE_API_KEY')
+llm = ChatGoogleGenerativeAI(model="gemini-pro",
+                             verbose = True,google_api_key=GOOGLE_API_KEY,
+                             convert_system_message_to_human=True)
+embeddings = GoogleGenerativeAIEmbeddings(model="models/embedding-001")
+
+# initialize chroma db
+vectordb=Chroma(embedding_function=embeddings, collection_name="resources")
```

Comment:
Reminder to persist the DB to disk so that we only have to ingest once, and can retrieve from other Python programs like `ingest_test.py`.